src/components/AssetsStatusAlert/index.jsx

```diff
@@ -0,0 +1,228 @@
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertUploadSuccess}
+            values={{ uploaded_count: (<FormattedNumber value={this.state.uploadSuccessCount} />) }}
+          />
+        );
+        alertType = 'success';
+        break;
+      case assetActions.upload.UPLOADING_ASSETS:
+        this.closeStatusAlert();
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertUploadInProgress}
+            values={{ uploading_count: (<FormattedNumber value={assetsStatus.count} />) }}
+          />
+        );
+        alertType = 'info';
+        break;
+      case assetActions.upload.UPLOAD_EXCEED_MAX_COUNT_ERROR:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertTooManyFiles}
+            values={{ max_count: (<FormattedNumber value={assetsStatus.maxFileCount} />) }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.upload.UPLOAD_EXCEED_MAX_SIZE_ERROR:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertTooMuchData}
+            values={{ max_size: (<FormattedNumber value={assetsStatus.maxFileSizeMB} />) }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.upload.UPLOAD_ASSET_FAILURE:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertGenericError}
+            values={{ assetName: assetsStatus.asset.name }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.lock.TOGGLING_LOCK_ASSET_FAILURE:
+        alertDialog = (
+          <WrappedMessage
+            message={messages.assetsStatusAlertFailedLock}
+            values={{ assetName: assetsStatus.asset.name }}
+          />
+        );
+        alertType = 'danger';
+        break;
+      case assetActions.clear.CLEAR_FILTERS_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.filter.FILTER_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.paginate.PAGE_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      case assetActions.sort.SORT_UPDATE_FAILURE:
+        alertDialog = genericUpdateError;
+        alertType = 'danger';
+        break;
+      default:
+        return;
+    }
+
+    this.setState({
+      statusAlertOpen: true,
+      statusAlertFields: {
+        alertDialog,
+        alertType,
+      },
+    });
+    this.statusAlertRef.focus();
+  }
+
+  closeDeleteStatus = () => {
+    this.closeStatusAlert();
+    this.props.onDeleteStatusAlertClose();
+  }
+
+  closeStatusAlert() {
+    this.props.clearAssetsStatus();
+
+    // clear out all status related state
+    this.setState({
```

Comment:
This looks the same as the default `state`? You could consider moving this into some constant instead of repeating?